src/App.tsx

```diff
@@ -1,52 +1,61 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
-
-// const tabs = [
-//   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
-//   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
-//   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
-// ];
+import { NavLink, Routes, Route } from 'react-router-dom';
+import { Tabs } from './components/Tabs';
 
 export const App = () => (
   <>
-    {/* Also requires <html class="has-navbar-fixed-top"> */}
     <nav
       className="navbar is-light is-fixed-top is-mobile has-shadow"
       data-cy="Nav"
     >
       <div className="container">
         <div className="navbar-brand">
-          <a href="/" className="navbar-item is-active">Home</a>
-          <a href="/tabs" className="navbar-item">Tabs</a>
+          <NavLink
+            to="/"
+            className={
+              ({ isActive }) => (
+                isActive
+                  ? 'navbar-item is-active'
+                  : 'navbar-item'
+              )
+            }
+          >
+            Home
+          </NavLink>
+
+          <NavLink
+            to="/tabs"
+            className={
+              ({ isActive }) => (
+                isActive
+                  ? 'navbar-item is-active'
+                  : 'navbar-item'
+              )
+            }
+          >
+            Tabs
+          </NavLink>
         </div>
       </div>
     </nav>
```

Comment:
Create new component for this nav section

Reply:
And add some paddings
![image](https://user-images.githubusercontent.com/57747481/188574693-f76addc3-8438-4258-a705-1f320ed4aa29.png)